Demo/src/androidTest/java/com/braintreepayments/demo/test/CardholderNameDropInTest.java

```diff
@@ -28,7 +28,7 @@ public void setup() {
     @Test(timeout = 60000)
     public void cardholderName_whenDisabled_isHidden() {
         setCardholderNameStatus(CardForm.FIELD_DISABLED);
-        onDevice(withText("Add Payment Method")).waitForExists().waitForEnabled().perform(click());
+        onDevice(withText("Add Payment Method")).waitForExists(20000).waitForEnabled(20000).perform(click());
```

Comment:
just curious, is there a reason we need to specify a wait time now?

Reply:
Yeah the `waitForExists()` method calls `waitForExists(5000)` under the hood. Increasing the timeout out to 20 seconds made the tests more stable for some reason.